public/js/actions/user.js

```diff
@@ -127,3 +127,25 @@ export function userSignOut(jquery=$) {
     });
   };
 }
+
+
+export function getBraintreeToken() {
+  console.log('Requesting braintree token');
+  return dispatch => {
+    $.ajax({
+      method: 'post',
+      url: '/api/braintree/token/generate/',
+      context: this,
+    }).then(data => {
+      dispatch({
+        type: actionTypes.RX_BRAINTREE_TOKEN,
+        user: {
+          braintreeToken: data.token,
+        },
```

Comment:
we generally keep data keys lower case with underscores. I guess that's a Python/API thing but it might be nice for consistency to call this `braintree_token`.